protocol-specs/generators/block_processing_transfers/index.js

```diff
@@ -2132,106 +90,142 @@ const accounts = {
 		balance: '0',
 		delegateName: 'genesis_100',
 	},
+	newAccount: {
+		passphrase:
+			'blame address tube insect cost knock major level regret bring april stick',
+		privateKey:
+			'b92e223981770c716ee54192a0ad028639d28d41221b72e455447bc4767aeb94caff2242b740a733daa3f3f96fc1592303b60c1704a8ac626e2704da039f41ee',
+		publicKey:
+			'caff2242b740a733daa3f3f96fc1592303b60c1704a8ac626e2704da039f41ee',
+		address: '2222471382442610527L',
+		balance: '0',
+	},
+	newAccount2: {
+		passphrase:
+			'female favorite client offer winner loud ostrich rich slogan jacket owner february',
+		privateKey:
+			'9b10f230fb2109cc77ee1a1ad659bf51f0e7b3c16697da416004dd409b427578ae88a228d37de7438f3d74df1713f3da00dd1c51afdc97bd47fa3aceb111a47a',
+		publicKey:
+			'ae88a228d37de7438f3d74df1713f3da00dd1c51afdc97bd47fa3aceb111a47a',
+		address: '11325618463998518034L',
+	},
 };
 
-const generateTestCasesForValidBlockWithNoTxs = () => {
-	const block = createBlock(
-		defaultConfig,
-		initialAccountState,
+const generateTestCasesValidBlockTransferTx = () => {
+	const chainStateBuilder = new ChainStateBuilder(
 		genesisBlock,
-		1,
-		0,
-		{
-			version: 1,
-			transactions: [],
-		},
+		initialAccountsState,
+		accounts,
 	);
 
+	chainStateBuilder
+		.transfer('50')
+		.from('16313739661670634666L')
+		.to('10881167371402274308L')
+		.forge();
+
+	chainStateBuilder
+		.transfer('40')
+		.from('10881167371402274308L')
+		.to('2222471382442610527L')
+		.forge();
+
+	const chainAndAccountStates = chainStateBuilder.getScenario();
+
 	return {
 		initialState: {
-			chain: [],
-			accounts: initialAccountState,
-		},
-		input: {
-			block,
+			chain: chainAndAccountStates.chain.slice(0, 1),
```

Comment:
Would be nice to add a comment why slice here?

Reply:
Added comments for this case and the other one using `-1`